push_ghcr.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+echo $GITHUB_TOKEN | docker login ghcr.io --username $GITHUB_USERNAME --password-stdin
+docker tag $timestamp_holder ghcr.io/$GITHUB_USERNAME/$timestamp_holder
+docker image push ghcr.io/$GITHUB_USERNAME/$timestamp_holder
+
+docker rmi $latest_holder
+docker rmi $timestamp_holder
+docker rmi ghcr.io/$GITHUB_USERNAME/$timestamp_holders
```

Comment:
This image is registry-specific, so probably worth keeping it here.

Reply:
Okay.